### DISCUSSION TITLE
Ideas for larger outreach

### DISCUSSION THREAD

**Discussion by shubhendumadhukar** [chryz-hub/chryz-hub.github.io#206] [Ideas]
Since this is a growing community, I believe we would have to actively seek new members and assist the initial growth. To that end I'd like to propose following items:
- More social platforms dedicated to the community. We are currently on Twitter but we can start and manage LinkedIn and Reddit accounts.
- A community calendar to show the upcoming events, next house meeting, roadmap discussion meetings.
- A dedicated YouTube channel which should be used for all community related activities
   - Live community meetings, where people can listen in on how the community works, what is the roadmap, what are the projects we are currently working on.
   - Monthly show and tell episode where we could give one community member a chance to showcase their project. This can even be extended to non members.
- Lastly, I think this is already being discussed in another thread, but newsletters and a blog website.

Since I am new to the community, I understand some of these might already be in the pipeline which I am not aware of.

**Comment by jomefavourite** [comment]
Very good idea shubhendumadhukar

**Comment by Evavic44** [comment]
Welcome to the community @shubhendumadhukar, thanks for the great suggestions. 
- Concerning the social media accounts for the community, Yes we are currently on twitter and discord for now as having multiple accounts for the community might be tougher to handle, though if you feel you can manage any of the accounts you mentioned, that'll be awesome. 😄
- Great idea on maximizing the community YouTube channel. These ideas will be strongly considered as we progress as a community. Though we decided to complete the community website first which would state our visions and goals as a community before working on any other projects as members have not been completely active; which is totally understandable. **_If you have any project ideas, do share with us so we can work on it._**
- Yes, as you are probably aware we are currently working on our community website and blog. 

_These are all great ideas and ways we can improve as a community. Absolutely awesome, 😎 thanks for raising them._

**Reply by chryzcode** [reply]
Ok, thanks everyone.
- Firstly, we actually need more social media handles to create awareness and introduce to people what we do.
 We actually had a team who does this stuff but it wasn't easy because they had other responsibility they needed to man in related to the community. So the task was much for them.
Having this accounts is all about responsibility and those who are actually committed to it. Because this stuff is also about been consistent, pushing out contents daily as related to the community and current progress and move taken in the community.
I'm into this  and I'm talking from experience,  even managing all our personal accounts at times might be difficult. So if you can help us out with this or you know anyone, please let the community be aware.

**Reply by shubhendumadhukar** [reply]
That's understandable. Maintaining multiple accounts is at times difficult. And in order to distribute the potential workload, I am willing to take up some platforms. I more inclined towards maintaining either YouTube or LinkedIn accounts, however I don't have a specific preference.

From a management perspective, we can have a discord channel which would be focus primarily on social media. An update could be sent to that channel by the admin team in case of any promotional event or even a generic post, which would then be picked by the individual platform leads and relayed on those platforms. Let me know your thoughts.

**Reply by chryzcode** [reply]
- Secondly, we have a [YouTube channel](https://youtube.com/channel/UCxro6LYOp3pmmuWDPMg-p1Q) where we teach tutorials, but due to some reasons it has been inactive. I once shared it out that of any member wants to create videos, we are all free but it would be reviewed. I can try to actually make a video about ChryzHub and post it on our YouTube page.

Making a live YouTube session can be some how hard now, because
- We need our members to be more active on the discord channel first.
- And an announcement has been made in the discord server. We would be holding a house call once in every two weeks, we can all check that out.
- I had an idea initially, using twitter spaces. We can actually use that to replace live sessions on YouTube for now. Where people beyond the members can join and follow along.

**Reply by chryzcode** [reply]
> That's understandable. Maintaining multiple accounts is at times difficult. And in order to distribute the potential workload, I am willing to take up some platforms. I more inclined towards maintaining either YouTube or LinkedIn accounts, however I don't have a specific preference.
> 
> From a management perspective, we can have a discord channel which would be focus primarily on social media. An update could be sent to that channel by the admin team in case of any promotional event or even a generic post, which would then be picked by the individual platform leads and relayed on those platforms. Let me know your thoughts.

Ok, I think a LinkedIn account would help. And I would try to make a channel also in which I did for the github and YouTube page for all social media accounts. So, that anything that posted would reflect on the discord server. 

Yeah, we have an announcement channel in which we let people know the direction in which we are driving towards. So anything we try to do, we make sure the community knows about it.
